### PR TITLE
Add wrappers to yast / Card 2671

### PR DESCRIPTION
**Pull request by NadjaW**

### CODE REVIEW

File: lib/suse/connect/yast.rb [hunk 1]
```diff
@@ -100,6 +100,23 @@ def system_migrations(products, client_params = {})
           Client.new(config).system_migrations(products)
         end
 
+        def all_products(client_params = {})
```
Comment (digitaltom) [inline lib/suse/connect/yast.rb]: I would prefer `activated_and_installed_products()` or `system_products()`

**Comment by digitaltom** [comment]
LGTM :8ball:

**Comment by wstephenson** [comment]
Anything holding up merging this?  I would like to see how it interacts with https://trello.com/c/RZBKlTUu/2677-3-migration-merge-provided-list-of-products-with-activated-products

File: lib/suse/connect/yast.rb [hunk 2]
```diff
@@ -100,6 +100,28 @@ def system_migrations(products, client_params = {})
           Client.new(config).system_migrations(products)
         end
 
+        # Returns installed and activated products on the system
+        # @param [Hash] client_params parameters to instantiate {Client}
+        # @return [Array <Connect::Product>] the list of system products
+        def system_products(client_params = {})
+          config = SUSE::Connect::Config.new.merge!(client_params)
+          status = Status.new(config)
```
Comment (vlewin) [inline lib/suse/connect/yast.rb]: you can squash it it one liner: `Status.new(config).system_products` like we do it in other methods for example: `Client.new(config).system_migrations(products)`

**Comment by vlewin** [comment]
LGTM, please check my latest comment

**Comment by lagartoflojo** [comment]
@wstephenson merged.

**Pull request merged**